### PR TITLE
First working version of FAB training for dipeptide

### PR DESCRIPTION
**Pull request by VincentStimper**
This is a first working version of a training script using FAB to approximate the Boltzmann distribution of the Alanine dipeptide. The results seem reasonable, although far away from perfect, e.g. the loss is increasing in the long run. However, this is something we can build upon.

### CODE REVIEW

**Review by lollcat — APPROVED** [review]
Nice!
